lib/Client.js

```diff
@@ -631,11 +631,18 @@ function pingCb(self) {
   self._handle.ping();
 }
 Client.prototype._ping = function() {
-  if (this._tmrInactive === undefined
+  if ((this._tmrInactive === undefined
       && typeof this._config.pingInactive === 'number'
+      && this._config.pingInactive > 0)
+      || (this._tmrInactive === undefined
       && typeof this._config.pingWaitRes === 'number'
-      && this._config.pingInactive > 0
-      && this._config.pingWaitRes > 0) {
+      && this._config.pingWaitRes > 0)) {
+    if (this._config.pingInactive === undefined){
```

Comment:
Missing space before `{` here and below.

Reply:
Fixed. Thanks.